app/models/wpcc/your_details_form.rb

```diff
@@ -6,11 +6,16 @@ class YourDetailsForm
     attr_accessor :age, :gender, :salary
     attr_accessor :salary_frequency, :contribution_preference
 
+    AGE = { minimum: 16, maximum: 75 }.freeze
```

Comment:
@Tr4pSt3R well, now your test description is wrong — "Scenario Outline: Older than the maximum age of 75 years" and yet the English part tests for 75 years exactly. 

My interpretation from the code (not from knowing anything about workplace pensions) would have been that 75 is the cut-off, so someone 74 years 364 days old is eligible, but the next day they're not. Which means the original "older than 74 years" was correct, but the code is testing "greater than or equal to 75". That's the numeric confusion, the words use one number, the code uses another. In my experience that's the kind of thing that causes problems later when someone makes an assumption based on one number and changes something that affects the other.

But if nobody else sees the slight discrepancy as a problem, revert that fixup and tell me to go jump :)